truss/templates/triton/model/utils/triton.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+
+def get_numpy_item(tensor):
+    """Returns the item of a numpy array as a Python type."""
+    item = tensor.as_numpy().item()
+    return item.decode("utf-8") if isinstance(item, bytes) else item
+
+
+def convert_tensor_to_python_type(tensor, dtype: str):
+    if dtype == "dict":
+        item = get_numpy_item(tensor)
+        return json.loads(item)
```

Comment:
what kind of guarantees do we have that this is in fact going to be formatted as json?

Reply:
We don't really. This is information that is being sent from the client so presumably, if they are sending a dict, it's a JSON serializable dict but this isn't enforced. I've updated this to reflect that.